writing-apps/our-first-app/the-build-system.md

```diff
@@ -69,6 +69,25 @@ If all went well, you should now be able to open your app from the Applications
 If you were about to add the "build" folder to your git repository and push it, stop! This binary was built for your computer and we don't want to redistribute it. In fact, we built your app in a separate folder like this so that we can easily delete or ignore the "build" folder and it won't mess up our app's source code.
 {% endhint %}
 
+## Uninstalling the application
+
+To uninstall your application, change into the build directory and we will use `ninja` once again to uninstall:
+
+```bash
+cd build
+sudo ninja uninstall
+```
+
+If all went well, you should see command output that shows files related to your application were removed.  Again, more details can be found in [Meson's documentation](https://mesonbuild.com/Manual.html).
+
+```bash
+...
+sudo rm /usr/bin/com.github.yourusername.hello-again
+sudo rm /usr/share/applications/com.github.yourusername.hello-again.desktop
+sudo rm /usr/share/metainfo/com.github.yourusername.hello-again.appdata.xml
+...
```

Comment:
The manual removal commands still appear in the copy, and the code block is without any context or explanation. I think that if you wanna keep the manual instructions for any reason at least provide context about what those commands do.

Reply:
Well, actually I was trying to give the user an indication of what they should see as output from the command, but I see how that is confusing and inconsistent. Most other bash code blocks are for the user to enter and run.

Reply:
This should be resolved by removing the bash output in https://github.com/elementary/docs/pull/134/commits/8ff32d85c43b45423fc92bb3ce95b455653d977a